README.md

```diff
@@ -4,7 +4,7 @@
 This project provides an Eclipse plugin for building, debugging, and deploying Google Cloud Platform applications.
 
 _TL;DR_: `mvn -Dtycho.toolchains=SYSTEM package` should
-generate a build into `gcp-repo/target/repository`.
+generate a p2-accessible repository in `gcp-repo/target/repository`.
```

Comment:
That's not correct: `gcp-repo` is for our built artifacts.  `ide-target-platform` is for our development use.